Formula/magicbell-cli.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class MagicbellCli < Formula
+  desc "Work with MagicBell from the command-line"
+  homepage "https://magicbell.com"
+  url "https://registry.npmjs.org/@magicbell/cli/-/cli-1.0.0.tgz"
+  sha256 "c5a64c61df069936a9a07609ea35f2d311405473ccb319ba21e2cf6f9bc26132"
+  license :cannot_represent # SEE LICENSE IN LICENSE
+
+  livecheck do
+    url "https://registry.npmjs.org/@magicbell/cli/latest"
+    regex(/["']version["']:\s*?["']([^"']+)["']/i)
+  end
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir[libexec/"bin/*"]
+  end
+
+  test do
+    assert_match(version.to_s, shell_output("#{bin}/magicbell --version 2>&1"))
+    assert_match(/you are not logged in/i, shell_output("#{bin}/magicbell logout 2>&1"))
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/magicbell --version")
    assert_match /you are not logged in/i, shell_output("#{bin}/magicbell logout")
```

Reply:
When I'd remove the braces, the second assert_match makes `brew audit` fail. Technically, I can remove the braces from the first line, but. having two different styles in a single block doesn't look pretty either.

I've removed the stderr redirect.

```
magicbell-cli
  * line 24, col 18: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.
```